app/schema/questionnaire.py

```diff
@@ -28,10 +28,13 @@ def add_group(self, group):
             group.container = self
 
     def get_item_by_id(self, item_id):
+        answer_without_instance_id = item_id.split('_')[0]
         if item_id == self.id:
             return self
         elif item_id in self.items_by_id.keys():
             return self.items_by_id[item_id]
+        elif answer_without_instance_id in self.items_by_id.keys():
+            return self.items_by_id[answer_without_instance_id]
```

Comment:
Removed this and stripping off the instance Id from the answer Id before this method is called.